deploy/kubernetes/metricbeat/README.md

```diff
@@ -2,30 +2,215 @@
 
 ## Ship metrics from Kubernetes to Elasticsearch
 
-### Kubernetes DaemonSet
+### Settings
 
-By deploying metricbeat as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)
-we ensure we get a running metricbeat daemon on each node of the cluster.
 
-Everything is deployed under `kube-system` namespace, you can change that by
-updating YAML manifests under this folder.
+We use official [Metricbeat Docker images](https://www.docker.elastic.co/r/beats/metricbeat), as they allow external files' configuration. Our YAML manifests are the following:
 
-### Settings
+<ul>
```

Comment:
I don't think I ever seen embedded html in our markdown readmes. do you mind removing it and replacing with markdown syntax? thanks

Reply:
Of course I can change that, but then I don't think I have a way to have multiple lines in one point of the list. I tried looking for a solution, but couldn't find any